TTHAnalysis/python/plotter/susy-ewkino/3l/functionsEWK.cc

```diff
@@ -324,13 +335,22 @@ int SR4l(int nTau, int nOSSF, int nOSLF, float mT2L, float mT2T, float mll, floa
     return 0;
 }
 
+
 int SR(int nLep, int nTau, int nOSSF, int nOSLF, float mT2L, float mT2T, float mll, float mT, float met) {
 
-    if(nLep == 3)
-        return SR3l(nTau, nOSSF, nOSLF, mT2L, mT2T, mll, mT, met);
-    if(nLep == 4)
-        return SR4l(nTau, nOSSF, nOSLF, mT2L, mT2T, mll, mT, met);
-    return 0;
+  if(nLep==2)
```

Comment:
Does this CR call need to be in the SR call? 
This is super dangerous to edit anything in this function (actually in the entire file) and I'd prefer to leave this function untouched and put it in a separate macro in the dedicated CR directory.